### PR TITLE
chore(Release notes): Update release notes for 2020.01

### LINKED ISSUE #3588: Update release notes for 2020.01

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #3588

**Additional issues**:

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -4,6 +4,95 @@ section: overview
...
 ---
 
+## 2020.01 release notes (2020-01-28)
+Packages released:
+
+
+### Components
+- **About modal:** 
+  - Added type safety to default props ([#3536](https://github.com/patternfly/patternfly-react/pull/3536))
+- **Actions column:**
```
Comment (redallen) [inline RELEASE-NOTES.md]: We have an `ActionsColumn` component? Shouldn't this be under `### Table`?

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -4,6 +4,95 @@ section: overview
...
+  - Properly assigned selectable and selected classes ([#3447](https://github.com/patternfly/patternfly-react/pull/3447))
+  - Removed hook ([#3434](https://github.com/patternfly/patternfly-react/pull/3434))
+  - Reverted React.Component back to React.FunctionComponent ([#3431](https://github.com/patternfly/patternfly-react/pull/3431))
+- **Data toolbar:** 
+  - Added and removed some modifiers to match core ([#3422](https://github.com/patternfly/patternfly-react/pull/3422))
+  - Reverted promotion of DataToolbar from experimental ([#3580](https://github.com/patternfly/patternfly-react/pull/3580))
+  - Updated example to not exclude previous filtered items. ([#3571](https://github.com/patternfly/patternfly-react/pull/3571))
+- **Divider:** 
+  - Promote Divider from experimental ([#3516](https://github.com/patternfly/patternfly-react/pull/3516))
+- **Flex layout:** Allowed string literals to be used for FlexItem modifiers ([#3544](https://github.com/patternfly/patternfly-react/pull/3544))
```
Comment (redallen) [inline RELEASE-NOTES.md]: Shouldn't this be under `### Layouts`?

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -4,6 +4,95 @@ section: overview
...
+- **Over flow menu:**
```
Comment (redallen) [inline RELEASE-NOTES.md]: Overflow is one word.

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -4,6 +4,95 @@ section: overview
...
+  - Upped version of gatsby-theme-patternfly-org ([#3562](https://github.com/patternfly/patternfly-react/pull/3562))
+- **Docs:** 
+  - Added a link to icons docs ([#3430](https://github.com/patternfly/patternfly-react/pull/3430))
+- **Extensions:** 
+  - Moved extensions to 'Extensions' in side nav category ([#3453](https://github.com/patternfly/patternfly-react/pull/3453))
+- **Integration:**
+  - Fixed selector in table simple actions ([#3584](https://github.com/patternfly/patternfly-react/pull/3584))
+- **Linting:** 
+  - Made Eslint fixes for *.md file ([#3493](https://github.com/patternfly/patternfly-react/pull/3493))
+  - Made initial eslint fixes for react-core-style-system *.md file ([#3505](https://github.com/patternfly/patternfly-react/pull/3505))
```
Comment (redallen) [inline RELEASE-NOTES.md]: Double-check your spacing around here, the Github editor stops syntax highlighting due to some inconsistent spacing I think.

File: RELEASE-NOTES.md (hunk 2 above)
Comment (tlabaj) [reply RELEASE-NOTES.md]: I debated that... I did not see a layout section (unless I missed it) in previous release notes where a layout was updated and I wanted to be consistent.

Comment (redallen) [reply RELEASE-NOTES.md]: It looks like we are not consistent about this. Core currently has a `### Layouts` as well as https://www.patternfly.org/v4/documentation/core/overview/release-notes `**Flex layout:**`. I'm okay with either after seeing that!

**Comment by tlabaj** [comment]
@redallen it is still missing the packages.  We are trying to get one more PR merged.

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -4,6 +4,96 @@ section: overview
...
+  - Fixed page layout imports for use with code sandbox.([#3461](https://github.com/patternfly/patternfly-react/pull/3461))
+- **Pagination:**
+  - Disabled navigation input if page is equal to zero ([#3472](https://github.com/patternfly/patternfly-react/pull/3472))
+- **Select:** 
+  - Add the type prop to the select toggle button ([#3495](https://github.com/patternfly/patternfly-react/pull/3495))
+  - Allowed custom option data comparison ([#3491](https://github.com/patternfly/patternfly-react/pull/3491))
+  - Updated so clear button appears with any type ahead input ([#3502](https://github.com/patternfly/patternfly-react/pull/3502))
+  - Set a unique id to the select toggle type ahead input ([#3529](https://github.com/patternfly/patternfly-react/pull/3529))
+- **Spinner** 
+ - Promote Spinner from experimental ([#3516](https://github.com/patternfly/patternfly-react/pull/3516))
```
Comment (rachael-phillips) [inline RELEASE-NOTES.md]: There's a small formatting error on spinner note. The bullet below needs to be indented.

File: RELEASE-NOTES.md [hunk 6]
```diff
@@ -4,6 +4,96 @@ section: overview
...
+
+### Catalog view extension
+- **Catalog tile:** Removed truncation fade and fixed length ([#3378](https://github.com/patternfly/patternfly-react/pull/3378))
```
Comment (rachael-phillips) [inline RELEASE-NOTES.md]: This is formatted differently than the other bullets

**Pull request merged**